app/src/main/java/com/pysiak/simple/clock/ui/TabAdapter.java

```diff
@@ -9,18 +9,19 @@
 import java.util.List;
 
 public class TabAdapter extends FragmentStatePagerAdapter {
-
     private final List<Fragment> mFragmentList = new ArrayList<>();
     private final List<String> mFragmentTitleList = new ArrayList<>();
 
     public TabAdapter(FragmentManager fm) {
```

Comment:
Check if needed - I think you could remove it

Reply:
Must be, because if not then android.support.v4.app.FragmentStatePagerAdapter does not have a default constructor .